packages/font-support-info/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@creately/font-support-info",
+  "version": "1.0.0",
+  "description": "This will check font support languages.",
```

Comment:
This package will be useful to check whether the given language is supported by the given font family. Returns true if it is supported.